exporter/debugexporter/internal/normal/metrics.go

```diff
@@ -0,0 +1,149 @@
+					dataPointLines = writeNumberDataPoints(metric, metric.Sum().DataPoints())
+				case pmetric.MetricTypeHistogram:
+					dataPointLines = writeHistogramDataPoints(metric)
+				case pmetric.MetricTypeExponentialHistogram:
+					dataPointLines = writeExponentialHistogramDataPoints(metric)
+				case pmetric.MetricTypeSummary:
+					dataPointLines = writeSummaryDataPoints(metric)
+				}
+				for _, line := range dataPointLines {
+					buffer.WriteString(line)
+				}
+			}
+		}
+	}
+	return buffer.Bytes(), nil
+}
+
+func writeNumberDataPoints(metric pmetric.Metric, dataPoints pmetric.NumberDataPointSlice) (lines []string) {
+	for i := 0; i < dataPoints.Len(); i++ {
+		dataPoint := dataPoints.At(i)
+		dataPointAttributes := writeAttributes(dataPoint.Attributes())
+
+		var value string
+		switch dataPoint.ValueType() {
+		case pmetric.NumberDataPointValueTypeInt:
+			value = fmt.Sprintf("%v", dataPoint.IntValue())
+		case pmetric.NumberDataPointValueTypeDouble:
+			value = fmt.Sprintf("%v", dataPoint.DoubleValue())
+		}
+
+		dataPointLine := fmt.Sprintf("%s{%s} %s\n", metric.Name(), strings.Join(dataPointAttributes, ","), value)
+		lines = append(lines, dataPointLine)
+	}
+	return lines
+}
+
+func writeHistogramDataPoints(metric pmetric.Metric) (lines []string) {
+	for i := 0; i < metric.Histogram().DataPoints().Len(); i++ {
+		dataPoint := metric.Histogram().DataPoints().At(i)
+		dataPointAttributes := writeAttributes(dataPoint.Attributes())
+
+		var value string
+		value = fmt.Sprintf("count=%d", dataPoint.Count())
+		if dataPoint.HasSum() {
+			value += fmt.Sprintf(" sum=%v", dataPoint.Sum())
+		}
+		if dataPoint.HasMin() {
+			value += fmt.Sprintf(" min=%v", dataPoint.Min())
+		}
+		if dataPoint.HasMax() {
+			value += fmt.Sprintf(" max=%v", dataPoint.Max())
+		}
+
+		for bucketIndex := 0; bucketIndex < dataPoint.BucketCounts().Len(); bucketIndex++ {
+			bucketBound := ""
+			if bucketIndex < dataPoint.ExplicitBounds().Len() {
+				bucketBound = fmt.Sprintf("le%v=", dataPoint.ExplicitBounds().At(bucketIndex))
+			}
+			bucketCount := dataPoint.BucketCounts().At(bucketIndex)
+			value += fmt.Sprintf(" %s%d", bucketBound, bucketCount)
+		}
+
+		dataPointLine := fmt.Sprintf("%s{%s} %s\n", metric.Name(), strings.Join(dataPointAttributes, ","), value)
+		lines = append(lines, dataPointLine)
+	}
+	return lines
+}
+
+func writeExponentialHistogramDataPoints(metric pmetric.Metric) (lines []string) {
+	for i := 0; i < metric.ExponentialHistogram().DataPoints().Len(); i++ {
+		dataPoint := metric.ExponentialHistogram().DataPoints().At(i)
+		dataPointAttributes := writeAttributes(dataPoint.Attributes())
+
+		var value string
+		value = fmt.Sprintf("count=%d", dataPoint.Count())
+		if dataPoint.HasSum() {
+			value += fmt.Sprintf(" sum=%v", dataPoint.Sum())
+		}
+		if dataPoint.HasMin() {
+			value += fmt.Sprintf(" min=%v", dataPoint.Min())
+		}
+		if dataPoint.HasMax() {
+			value += fmt.Sprintf(" max=%v", dataPoint.Max())
+		}
+
+		// TODO display buckets
```

Comment:
Can we file an issue for this?

Reply:
Done: https://github.com/open-telemetry/opentelemetry-collector/issues/10463